src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -96,6 +98,7 @@ public class GuiConfigController implements Initializable {
   private static final String HINTS_VOTER_ERROR_RULES_FILENAME =
       "network/brightspots/rcv/hints_voter_error_rules.txt";
   private static final String HINTS_OUTPUT_FILENAME = "network/brightspots/rcv/hints_output.txt";
+  private static final String CVR_FILE_PATH_DELIMITER = ";";
```

Comment:
On macOS/Linux, it's possible to have a semicolon in a filename. :/

Reply:
Same in Windows, actually, I just chose something that I thought was unlikely. It seems the only illegal character for *nix is /, which is obviously not viable to use as a delimiter. I changed to to `;;` to make it less likely to interfere with a legit file path.